test-apps/kitchen/bin/kitchen

```diff
@@ -554,7 +556,17 @@ class Kitchen(HTTPWebSocketsHandler):
                     chunk = bytes(itertools.islice(response_bytes, self.__chunk_size))
                     if len(chunk) == 0:
                         if self.__chunked:
-                            self.wfile.write(b'0\r\n\r\n')
+                            self.wfile.write(b'0\r\n')
+                            if self.__trailer_delay_secs > 0:
```

Comment:
I verified with curl that this mechanism does indeed send the 0, sleeps and then sends the trailers.

I was unable to mimic the same behavior with a [Netty support for HTTP2 server](https://netty.io/4.1/api/io/netty/handler/codec/http2/package-summary.html). With the following code:
```
    private class EchoTrailersResponder extends RequestResponder {
        @Override
        void onHeadersRead(ChannelHandlerContext ctx, int streamId, boolean endOfStream, Http2Headers headers) {
            encoder().writeHeaders(ctx, streamId, createDefaultResponseHeaders(), 0, false, ctx.newPromise());
            ctx.flush();
            Thread.sleep(10000);

            encoder().writeData(ctx, streamId, RESPONSE_BYTES.duplicate(), 0, false, ctx.newPromise());
            ctx.flush();
            Thread.sleep(10000);

            Http2Headers responseTrailers = createResponseHeadersFromRequestHeaders(headers).add("trailer", "value1", "Value2");
            encoder().writeHeaders(ctx, streamId, responseTrailers, 0, true, ctx.newPromise());
            ctx.flush();
        }
    }
```
I saw in curl that we would receive the headers and then sleep 20 secs before receiving the body and trailers together. I verified that making requests with a jet client to this server received the body and headers though.